authui/src/authflowv2/components/input.css

```diff
@@ -32,15 +33,20 @@
     );
 
     --input__password-padding-end: 2.375em; /* 38 / 16 = 2.375 */
+    --input__password-icon-color: var(--color-neutral-500);
```

Comment:
Lets name it ``--input__password-visibility-icon-color` which seems more clear what is this icon.